.github/workflows/create-release.yaml

```
@@ -0,0 +1,82 @@
+name: Create Release
+run-name: Create Release ${{ inputs.name }}
+
+env:
+  IMAGE_REPO: europe-docker.pkg.dev/kyma-project/prod/eventing-publisher-proxy
+
+on: workflow_dispatch
+
+jobs:
+  verify-release:
+    name: Verify image version
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout EPP repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Verify that the current branch has a name that starts with 'release-'
+        run: |
+          CURRENT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+          if [[ "$CURRENT_BRANCH" == release-* ]]; then
+            echo "current_branch=$CURRENT_BRANCH" >> $GITHUB_ENV
+            echo "Branch name starts with 'release-'."
+          else
+            echo "Branch name does not start with 'release-'."
+            exit 1
+          fi
+      - name: Get the next release version
```

for consistentcy with our other gh workflows, can you please put empty lines in between steps?
```suggestion
          fi
          
      - name: Get the next release version
```

---

Done